### DISCUSSION TITLE
Trouble with LFS

### DISCUSSION THREAD

**Discussion by shruti-sriniv** [community/community#48156] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I am trying to install LFS as per the github directions, but when I try to run ./install.sh while in the cd git-lfs-3.3.0, then it shows me an error that says "Operation not permitted". Could someone help me out with this and tel me why I am getting this error and how to fix it?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @shruti-sriniv ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
